docs/docs/30-administration/00-setup.md

```diff
@@ -171,6 +171,16 @@ services:
 +     - WOODPECKER_AGENT_SECRET=${WOODPECKER_AGENT_SECRET}
 ```
 
+### Kubernetes 
+
+We recommended to deploy Woodpecker using the [Woodpecker helm chart](https://github.com/woodpecker-ci/helm).
+Have a look at the `values.yaml` config files for all available settings.
```

Comment:
We could link the values file here

Reply:
I'd prefer the top level link - as the sentence is before the reference to `values.yml`.

Reply:
I would leave the top-level one as well, but add another one like this for the values file:

```suggestion
Have a look at the [`values.yaml`](https://github.com/woodpecker-ci/helm/blob/main/values.yaml) config files for all available settings.
```